### ISSUE TITLE
Update nomenclature before we get popular.

### ISSUE THREAD

**Issue by Leopere** [ohmyform/ohmyform#7]
Ideally, we should be avoiding getting stuck with TellForm names in any of the examples or database structure long term so if we can sift through with a find & replace function and remove everything without removing direct references to the upstream.

**Comment by kiwicopple** [comment]
I don't know if this is worth opening another issue, but does this mean that this fork is ready to use? If I'm starting now, should I use ohmyform or tellform?

This looks cool btw!

**Comment by wodka** [comment]
@kiwicopple you should definitely use OhMyForm.

**Comment by kiwicopple** [comment]
Thanks @wodka 👍

**Comment by Leopere** [comment]
@kiwicopple how did you generate your GitHub photo btw?

**Comment by kiwicopple** [comment]
@Leopere  I used the Primitive software by Michael Fogleman: 
https://www.michaelfogleman.com/#primitive

Probably the only thing I have used it for, but his projects are all so great I wanted to support him by purchasing something from him

**Comment by karser** [comment]
Looks like this is related
![image](https://user-images.githubusercontent.com/1675033/64102442-ac2e5d00-cd78-11e9-9e48-c10408db2ba6.png)

**Comment by wodka** [comment]
thx @karser will change this

**Comment by karser** [comment]
I guess it needs massive replacement

![image](https://user-images.githubusercontent.com/1675033/64104751-83f52d00-cd7d-11e9-8d1a-23d02c6ac030.png)

![image](https://user-images.githubusercontent.com/1675033/64104901-da626b80-cd7d-11e9-800a-9cabd96004d0.png)

Also there is sentry dsn points to tellform: `Raven.config('https://825fefd6b4ed4a4da199c1b832ca845c@sentry.tellform.com/2')`

**Comment by Leopere** [comment]
We can re-open this until the rest of these things are cleaned up.  I imagine the next big release from @wodka will be able to close this off.

**Comment by wodka** [comment]
just checked in my branch - I cannot find the references there - will also check the next build if this is still existing

**Comment by vickywane** [comment]
Hi , 

I would like to make some contributions here as part of the **hacktoberfest**.

I read the issue description but couldnt understand it clearly. 

Do you mind explaining the objective @leopere ?

**Comment by Leopere** [comment]
Hey @vickywane this one should be pretty simple for the most part we just need to eliminate naming references to TellForm in the code as we're attempting to eventually replace the entire codebase with something a bit more modern.  If you find any instances or references of TellForm that's useful to us.

**Comment by vickywane** [comment]
Alright . So i'll just make a PR stating where i found them and snapshots .

**Comment by vickywane** [comment]
I created this https://github.com/tellform/tellform/pull/349 . 
Is this a good starting point ? 

@wodka @Leopere

**Comment by wodka** [comment]
@vickywane Not really, we want to remove all remaining references of tellform, because ohmyform is the way to go forward.

The best way is to start it locally and check at least the screenshots from this thread to remove tellform

**Comment by vickywane** [comment]
Great. 

It seems i didn't understand the task clearly . 

I'll try run the project locally

**Comment by Leopere** [comment]
That’s no problem at all I appreciate the efforts! Technically that PR does
work for Hacktoberfest however we are looking to find the instances of the
word TellForm in the OhMyForm codebase that weren’t caught before in our
previous removal attempt. I still want to leave credit to it but this
codebase will be very different in the future from TellForm.

**Comment by vickywane** [comment]
Alright . 
Am willing to spend some more time to contributing to that . 

I had some troubles trying to run the docker images locally yesterday . 
Would it be ideal to run some sort of text search operation on  the code-base ?

**Comment by Leopere** [comment]
Do whatever seems appropriate just make sure that you test your build and
that there are no regressions from your changes.
